static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -147,7 +147,17 @@ define([
             }
         }
 
-        function showBanner(params) {
+        var paypalAndCreditCardImage = (function() {
```

Comment:
have a look at https://github.com/guardian/frontend/pull/16576

and https://github.com/guardian/frontend/blob/fix-epic-in-liveblog-campaign-code/static/test/javascripts-legacy/setup.js#L32